src/content/developers/docs/networking-layer/network-addresses/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Network addresses
+description: An introduction to network addresses.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Ethereum nodes have to identify themselves with some basic information to connect to peers. To ensure any potential peer can interpret this information, it is relayed in one of three standardized formats that any Ethereum node can understand: multiaddr, enode, or Ethereum Node Records (ENRs). ENRs are the current standard for Ethereum network addresses.
+
+## Prerequisites {#prerequisites}
+
+Some understanding of Ethereum's [networking layer](/src/content/developers/docs/networking-layer/) is required to understand this page.
+
+## Multiaddr {#multiaddr}
+
+The original Ethereum node address format was the 'multiaddr' (short for 'multi-addresses'). Multiaddr is a universal format designed for peer-to-peer networks. Addresses are represented as key-value pairs with keys and values separated with a forward slash. For example, the multiaddr for a node with IPv4 address `192.168.22.27` listening to TCP port `33000` looks like:
+
+`/ip6/192.168.22.27/tcp/33000`
+
+For an Ethereum node, the multiaddr contains the node-ID (a hash of their public key):
+
+`/ip6/192.168.22.27/tcp/33000/p2p/5t7Nv7dG2d6ffbvAiewVsEwWweU3LdebSqX2y1bPrW8br`
+
+## Enode {#enode}
+
+An enode is a way to identify an Ethereum node using a URL address format. The hexadecimal node-ID is encoded in the username portion of the URL separated from the host using an @ sign. The hostname can only be given as an IP address; DNS names are not allowed. The port in the hostname section is the TCP listening port. If the TCP and UDP (discovery) ports differ, the UDP port is specified as a query parameter "discport"
+
+In the following example, the node URL describes a node with IP address `10.3.58`, TCP port `30303` and UDP discovery port `30301`.
```

Comment:
And is this a valid IP address?

Reply:
Right but IP addresses have four fields 🙂 🤔 
https://www.ibm.com/docs/en/ts3500-tape-library?topic=functionality-ipv4-ipv6-address-formats

Reply:
IPv4 at least

Reply:
lol aha ok!